src/minidriver/minidriver.c

```diff
@@ -1402,7 +1402,7 @@ md_fs_add_msroot(PCARD_DATA pCardData, struct md_file **head)
 	for(ii = 0; ii < cert_num; ii++)   {
 		struct sc_pkcs15_cert_info *cert_info = (struct sc_pkcs15_cert_info *) prkey_objs[ii]->data;
 		if (cert_info->authority) {
-			dwret = md_fs_add_file(pCardData, head, "msroot", EveryoneReadUserWriteAc, NULL, 0, NULL);
+			dwret = md_fs_add_file(pCardData, head, "msroots", EveryoneReadUserWriteAc, NULL, 0, NULL);
```

Comment:
@hamano is right: in all my projects, I'm using either szROOT_STORE_FILE or directly the string "msroots".  I may have done the typo in my test program and then reported it in the source code. Sorry.